src/ucp/api/ucp.h

```diff
@@ -1188,6 +1196,18 @@ typedef struct ucp_worker_params {
      */
     uint64_t                flags;
 
+    /**
+     * Worker name is intended for identification of the worker during tracing
+     * and analysis of UCX-based applications.
+     * This value is optional.
+     * The actual name of the worker can be obtained by @ref ucp_worker_query
```

Comment:
The library can change the name provided by user in order to avoid potential name conflicts. It is advise to query the name using @ref ucp_worker_query

Reply:
The library can change the name provided by a user to avoid potential name conflicts. Please use `@ref  ucp_worker_query` to get the actual worker name.

@shamisp @tonycurtis is this one ok?